### PR TITLE
Add iOS 9.3 support and update donation options

### PR DESCRIPTION
**Pull request by palxex**
Adds iOS 9.3 support, should also works for iOS 7/8/9.0-9.1 ( not tested due to lack of device ), not sure for iOS 9.2. Previous design only for iOS9, so pangu8 protection was replaced to pangu 9.2-9.3 protection.

### CODE REVIEW

File: CyDelete.xm [hunk 1]
```diff
@@ -325,59 +358,93 @@ static void uninstallClickedForIcon(SBIcon *self) {
...
+			//new mechanism: iOS 9.2 above will not automatically trigger uninstall, we've to call it manually
+			BOOL contains = NO;
+			for(CDUninstallDpkgOperation *operation in [uninstallQueue operations]){
+				if([operation.package isEqualToString:[app bundleIdentifier]]){
+					contains = YES;
+					break;
+				}
+			}
+			if(contains) {
+				UIAlertController* alert=[UIAlertController alertControllerWithTitle:[NSString stringWithFormat:@"%@ IS uninstalling",[app displayName]] message:@"Please WAIT A SEC" preferredStyle:UIAlertControllerStyleAlert];
```
Comment (ryanb93) [inline CyDelete.xm]: Lets make this text a little bit more professional.

`UIAlertController* alert=[UIAlertController alertControllerWithTitle:[NSString stringWithFormat:@"%@ is uninstalling",[app displayName]] message:@"Please wait. This should only take a few seconds." preferredStyle:UIAlertControllerStyleAlert];`

File: CyDelete.xm [hunk 2]
```diff
@@ -397,11 +463,17 @@ static void uninstallClickedForIcon(SBIcon *self) {
 	}
 
 	-(NSString *)uninstallAlertTitle {
-		return [NSString stringWithFormat:SBLocalizedString(@"UNINSTALL_ICON_TITLE"),
+		%log;
```
Comment (ryanb93) [inline CyDelete.xm]: remove this %log

File: CyDelete.xm [hunk 3]
```diff
@@ -397,11 +463,17 @@ static void uninstallClickedForIcon(SBIcon *self) {
...
+		if(!_forceCydia && isOfficialUninstallable([self application]))
+			return %orig;
+		return [NSString stringWithFormat:cydelete_IOSVersionAbove92() ? SBLocalizedString(@"UNINSTALL_ICON_TITLE_DELETE_WITH_NAME") : SBLocalizedString(@"UNINSTALL_ICON_TITLE"),
 						[[self application] displayName]];
 	}
 
 	-(NSString *)uninstallAlertBody {
+		%log;
```
Comment (ryanb93) [inline CyDelete.xm:460]: remove log

File: CyDelete.xm [hunk 4]
```diff
@@ -427,11 +499,17 @@ static void uninstallClickedForIcon(SBIcon *self) {
 	}
 
 	-(NSString *)uninstallAlertConfirmTitle {
-		return SBLocalizedString(@"UNINSTALL_ICON_CONFIRM");
+		%log;
```
Comment (ryanb93) [inline CyDelete.xm]: remove log

File: CyDelete.xm [hunk 5]
```diff
@@ -427,11 +499,17 @@ static void uninstallClickedForIcon(SBIcon *self) {
...
+		if(!_forceCydia && isOfficialUninstallable([self application]))
+			return %orig;
+		return cydelete_IOSVersionAbove92() ? SBLocalizedString(@"UNINSTALL_ICON_BUTTON_DELETE") : SBLocalizedString(@"UNINSTALL_ICON_CONFIRM");
 	}
 
 	-(NSString *)uninstallAlertCancelTitle {
-		return SBLocalizedString(@"UNINSTALL_ICON_CANCEL");
+		%log;
```
Comment (ryanb93) [inline CyDelete.xm]: remove log

File: README.md [hunk 6]
```diff
@@ -1,4 +1,4 @@
 CyDelete
 =========
 
-The original mobile substrate CyDelete tweak created by @DHowett updated for iOS 7 and 8. 
+The original mobile substrate CyDelete tweak created by @DHowett and ported to iOS7/8 by Ryan Burke updated for iOS 9.
```
Comment (ryanb93) [inline README.md]: A mobile substrate tweak to uninstall applications from the homescreen that have been installed using Cydia. 

### Contributors
CyDelete - [Dustin Howett ](https://github.com/DHowett)
CyDelete 7 & 8 - [Ryan Burke](https://github.com/ryanb93)
CyDelete 9 - [Pal Lockhart](https://github.com/palxex)

File: CyDelete/Resources/CyDelete.plist [hunk 7]
```diff
@@ -68,7 +68,7 @@
 		</dict>
 		<dict>
```
Comment (ryanb93) [inline CyDelete/Resources/CyDelete.plist]: You're missing entries here to show your donation links for Bitcoin and AliPay :)

**Review by ryanb93 — CHANGES_REQUESTED** [review]

File: CyDelete.xm (hunk 1 above)
Comment (ryanb93) [reply CyDelete.xm]: In addition to this, we should probably be using localized strings here.

**Comment by palxex** [comment]
In fact seems this dialog not need to show in latest build. Previous I’m using `apt-get remove then uicache` way, and it disappears too late( almost 10 secs after clicking) so I added the dialog. Then I changed to the new way, first unregister app to hide the icon, then apt-get remove in background; this way the dialog seems unnecessary, since user will have no time on click it again, but I forgot to remove the code. Let me modify it tomorrow :P too late in me time region, almost sun rising, too easy to introduce bug.

**Pull request closed without merging**